hikey.mk

```diff
@@ -415,10 +416,11 @@ recovery:
 	@echo
 	$(call flash_help)
 	@echo
-	python $(ROOT)/burn-boot/hisi-idt.py --img1=$(LLOADER_PATH)/l-loader.bin
+	python $(ROOT)/burn-boot/hisi-idt.py --img1=$(LLOADER_PATH)/recovery.bin
+	fastboot flash loader $(LLOADER_PATH)/l-loader.bin
 	@echo
 	@echo "3. Wait until you see the (UART) message"
-	@echo "    \"Enter downloading mode. Please run fastboot command on Host.\""
+	@echo "    \"Enter fastboot mode. Please run fastboot command on Host.\""
```

Comment:
It's not the message I'm seeing. After booting with the jumpers in recovery mode, and running the Python script, the console is like this:
```
NOTICE:  Booting Trusted Firmware
NOTICE:  BL1: v1.4(release):v1.5-rc2
NOTICE:  BL1: Built : 10:05:55, Mar  9 2018
NOTICE:  BL1-FWU: *******FWU Process Started*******
NOTICE:  Booting Trusted Firmware
NOTICE:  BL1: v1.1(release):af5ddb1
NOTICE:  BL1: Built : 10:04:04, Mar  9 2018
NOTICE:  syspll frequency:1190494208Hz
NOTICE:  succeed to init lpddr3 rank0 dram phy
INFO:    lpddr3_freq_init, set ddrc 533mhz
INFO:    init ddr3 rank0
INFO:    ddr3 rank1 init pass
INFO:    lpddr3_freq_init, set ddrc 800mhz
INFO:    init ddr3 rank0
INFO:    ddr3 rank1 init pass
INFO:    Elpida DDR
NOTICE:  Enter fastboot mode...
```

And AFAICT there is no need to wait. For me at least I can run the fastboot commands immediately, they'll just wait for the board to be ready if it's not.

Reply:
Updated UART message. Thanks.